projects/canopy/src/lib/breadcrumb/breadcrumb-item/breadcrumb-item.component.scss

```
@@ -16,6 +16,16 @@
     }
   }
 
+  &__container--hide-icons {
+    .lg-breadcrumb-item__content .lg-icon {
```

just wondering if you need `.lg-breadcrumb-item__content .lg-icon` or could you get rid of `.lg-breadcrumb-item__content`? No biggie, but it might reduce the specificity and is a tiny bit less code/
```suggestion
    .lg-icon {
```

---

the content icons are those set using content projection i.e. the home icon in this example. I think the proposed change would affect the chevrons which are also icons.

---

make sense 👍